palautteet/hairinta/views.py

```diff
@@ -0,0 +1,14 @@
+from ..email import EmailService
+from ..views import get_form_view
+from .forms import HairintaForm
+
+
+TEMPLATE = 'hairintailmoitus.html'
+EMAIL_SERVICE = EmailService('Häirintäilmoitus',
+                             'hairinta@satakuntatalo.fi',
+                             ['kuraattori@satakuntatalo.fi'])
```

Comment:
If the list of parameters gets too large, I have started the list from a new line. If the name of the function changes, the other lines don't have to be realigned.
```
EMAIL_SERVICE = EmailService(
    'Häirintäilmoitus',
    'hairinta@satakuntatalo.fi',
    ['kuraattori@satakuntatalo.fi']
)
```

Reply:
Good point. We should get a linter (and an autoformatter such as `black` as well) :P